app/graph/types/comment_type.rb

```diff
@@ -3,6 +3,9 @@
   description 'A comment a user left on some content.'
   global_id_field :id
 
-  field :author, !UserType, property: :user
+  field :body, !types.String
+  field :author, UserType
```

Comment:
Should this be nullable?

Reply:
yup

Reply:
Ops, I misread.
It should be non-nullable.